### PR TITLE
Add steps for enabling Insider version of Edge for WebNN Dev Preview

### PR DESCRIPTION
**Pull request by ibelem**
@fdwr @Adele101 PTAL 

Fully copied from https://blogs.windows.com/windowsdeveloper/2024/08/29/directml-expands-npu-support-to-copilot-pcs-and-webnn/

@mingmingtasd @huningxin

127 0 0 1_8080_install html

### CODE REVIEW

File: install.html [hunk 1]
```diff
@@ -206,25 +206,69 @@
...
+        </li>
+        <li>
+          Download DirectML redistributable
+          <ol style="list-style: lower-alpha;">
+            <li>Download DirectML from <a href="https://www.nuget.org/packages/Microsoft.AI.DirectML/1.15.2">https://www.nuget.org/packages/Microsoft.AI.DirectML/1.15.2</a></li>
+            <li>Rename <code>microsoft.ai.directml.1.15.2.nupkg</code> to <code>microsoft.ai.directml.1.15.2.nupkg.zip</code> and extract it</li>
+          </ol>
+        </li>
+        <li>
+          Copy <code>microsoft.ai.directml.1.15.2.nupkg.zip\bin\-win\directml.dll</code> to the appropriate directory<br>(replace with x64 on Intel devices and arm64 on Qualcomm devices)
```
Comment (fdwr) [inline install.html:244]:
```suggestion
          Copy <code>microsoft.ai.directml.1.15.2.nupkg.zip\bin\&lt;arch&gt;-win\directml.dll</code> to the appropriate directory<br>(replace with x64 on Intel devices and arm64 on Qualcomm devices)
```

**Review by fdwr — APPROVED** [review]
Small typo, else LGTM. Thank you Belem.

**Comment by fdwr** [comment]
What do you think about keeping the simple instructions for the GPU case, and relegating all this extra complexity to the NPU case (the instructions from that blog post were specifically about NPU anyway)? So there could be a new heading "Enabling Insider version of Edge for WebNN Dev Preview NPU"?

**Comment by ibelem** [comment]
Thanks @fdwr for the fix and feedback, added GPU steps back, PTAL

127 0 0 1_8080_install html (1)

**Review by fdwr — APPROVED** [review]
👍 Yeah, then it also makes it clear that the minimum OS version differs between GPU vs NPU.

**Pull request merged**